dlk/python/dlk/templates/src/func/generic/batch_normalization.cpp

```diff
@@ -20,32 +20,20 @@ limitations under the License.
 #include "func/batch_normalization.h"
 #include "time_measurement.h"
 
-static const auto scale = std::make_unique<float[]>(MAX_IN_C);
-static const auto shift = std::make_unique<float[]>(MAX_IN_C);
-
 void func_BatchNormalization(const TensorView<T_FLOAT, MemoryLayout::NHWC>& input,
```

Comment:
For consistency, it might be better to rename this function to `func_BatchNormalizationOptimized`. What do you think?

Reply:
Renamed.

Reply:
thanks!